starcheck/src/starcheck.pl

```diff
@@ -48,6 +48,7 @@
 use Inline Python => q{
 
 import os
+import traceback
```

Comment:
Unused import?

Reply:
This one will get used in the kadi PR (which also wraps get_ccd_temps) so I didn't care about cutting it.